src/devtools/client/debugger/src/components/Editor/LineHitCounts.tsx

```diff
@@ -90,12 +90,15 @@ function LineHitCounts({ editor, isCollapsed, setIsCollapsed }: Props) {
         let className = styles.HitsBadge0;
         let index = null;
         if (hitCount > 0) {
```

Comment:
Oh, that's a fun edge case. Should index be `NUM_GRADIENT_COLORS - 1` in this case?